warn symbol is part of libc.o, then i have to look for a
different name
  Alejandro

--- ./src/complain.c.orig	2004-02-11 19:14:26.000000000 -0300
+++ ./src/complain.c	2004-02-11 19:16:19.000000000 -0300
@@ -87,7 +87,7 @@
 }
 
 void
-warn (const char *message, ...)
+warn_this (const char *message, ...)
 {
   va_list args;
 
--- ./src/complain.h.orig	2004-02-11 19:14:33.000000000 -0300
+++ ./src/complain.h	2004-02-11 19:16:32.000000000 -0300
@@ -27,7 +27,7 @@
 
 /* Informative messages, but we proceed.  */
 
-void warn (char const *format, ...)
+void warn_this (char const *format, ...)
   __attribute__ ((__format__ (__printf__, 1, 2)));
 
 void warn_at (location loc, char const *format, ...)
--- ./src/files.c.orig	2004-02-11 19:14:43.000000000 -0300
+++ ./src/files.c	2004-02-11 19:16:41.000000000 -0300
@@ -333,5 +333,5 @@
   for (j = 0; j < names; j++)
     for (i = 0; i < j; i++)
       if (strcmp (name[i], name[j]) == 0)
-	warn (_("conflicting outputs to file %s"), quote (name[i]));
+	warn_this (_("conflicting outputs to file %s"), quote (name[i]));
 }
